centinel/backend.py

```diff
@@ -29,7 +30,8 @@ def request(self, slug):
         url = "%s/%s" % (self.config['server']['server_url'], slug)
         try:
             req = requests.get(url, auth=self.auth,
-                               proxies=self.config['proxy']['proxy'])
+                               proxies=self.config['proxy']['proxy'],
+                               verify=self.verify)
```

Comment:
When I was removing the certificate bundle, I removed the `verify` argument here which enables verification by default. I've added it back so that we can set it to `False`.
It made some lines too long, so I just made it a class attribute.